app/views/forms/_screen.html.erb

```diff
@@ -37,8 +37,101 @@
 
     </div>
 
-    <%# questions on the form %>
-    <%= index_table(Questioning, @form.questionings, :within_form => true) if form_mode != :new %>
+    <div class="questions">
+      <input class="btn btn-primary" type="submit" value="Save"></input>
+
+      <h2>Questions</h2>
+
+      <div class="questions-header" id="questions-header">
+        <span><a href="choose-questions"><i class="fa fa-plus"></i> Add Question</a></span>
+        <span class="q-addGroup"><a><i class="fa fa-folder-open-o"></i> Add Group</a></span>
```

Comment:
* I think the span can be removed here too. Just put the class on the `<a>`?
* I think you need to have `href=#` in the `<a>` or it won't work right.
* For class names, please use dash-erized instead of camelCase.
* 'Add Group' should go in the en.yml file. Please put it under the forms key. Search the file for 'add_questions' and you'll see where I mean.